_data/projects/Learning_Object_Oriented_Python.yml

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
@josharsh Add it back, you made a change to some other file

Reply:
Didn't quite get what happened. I tried resolving the conflicts. I mistakenly used a single commit initially which affected both files. I am stuck now!

Reply:
I can't see extra lines added onto the files into my repository, and it looks fine on syntax.

Reply:
> I am stuck now!

I'd just use the web editor to [create a new file](https://help.github.com/en/articles/creating-new-files) against the default branch.